apis/Google.Cloud.Spanner.Data/docs/connection_string.md

```diff
@@ -175,4 +175,14 @@ The valid type mappings for `SpannerDbType.Date` are:
 Examples:
 
 - SpannerToClrTypeDefaultMappings=DateToDateTime
-- SpannerToClrTypeDefaultMappings=DateToSpannerDate
\ No newline at end of file
+- SpannerToClrTypeDefaultMappings=DateToSpannerDate
+
+## DatabaseRole
+
+The database role associated to this connection string. This option is only used if Fine Grained
```

Comment:
associated to => associated with - or just "Option to configure the database role for this connection." (That would be more in-keeping with the options above.)